### ISSUE TITLE
default node-config.yaml can't parse

### ISSUE THREAD

**Issue by andrewarrow** [input-output-hk/jormungandr#3235]
Hello I've been trying to follow:

https://github.com/Chris-Graffagnino/Jormungandr-for-Newbs/blob/master/docs/jormungandr_node_setup_guide.md

or

https://github.com/Kevmate/Jormungandr-for-Newbs/blob/master/docs/jormungandr_node_setup_guide.md

but I can never get this yaml file:

```
log:
- output: stderr
  format: plain
  level: info
p2p:
  topics_of_interest:
    blocks: high
    messages: high
  listen_address: "/ip4/0.0.0.0/tcp/3000"
  public_address: "/ip4/161.35.232.233/tcp/3000"
  public_id: 1dec8c7f307d04fa0e3753b0d560857a20a3afc6a3e48cba
  max_connections: 1024
  gossip_interval: 10s
  trusted_peers:
    - address: "/ip4/13.56.0.226/tcp/3000"
      id: 7ddf203c86a012e8863ef19d96aabba23d2445c492d86267
    - address: "/ip4/54.183.149.167/tcp/3000"
```

to parse right.

errors like:

|-> Error while parsing the node configuration file: log[0]: invalid type: map, expected a string at line 2 column 9
 |-> log[0]: invalid type: map, expected a string at line 2 column 9

but something seems wrong since I didn't make any changes yet. I'm just trying to start up my Jormungandr node for the very first time. But there doesn't seem to be any example node-config.yaml that just works out of the box?

**Comment by andrcmdr** [comment]
Here's the documentation about node configuration parameters:

https://input-output-hk.github.io/jormungandr/quickstart/02_passive_node.html#the-node-configuration

https://input-output-hk.github.io/jormungandr/configuration/introduction.html#node-configuration

Same in repo:

https://github.com/input-output-hk/jormungandr/blob/master/doc/quickstart/02_passive_node.md#the-node-configuration

https://github.com/input-output-hk/jormungandr/blob/master/doc/configuration/introduction.md#node-configuration

Recently network p2p configuration and some other things in config was extended.
Thus, more recent detailed information about configuration options can be found here - look at the data structure fields and their default implementations:
https://github.com/input-output-hk/jormungandr/blob/master/jormungandr/src/settings/start/config.rs#L18

and here:
https://github.com/input-output-hk/jormungandr/tree/master/jormungandr-lib/src/interfaces/config

Configuration file entry point:
https://github.com/input-output-hk/jormungandr/blob/master/jormungandr/src/settings/command_arguments.rs#L24


Configuration produced in tests also could be really helpful:

https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-testing-utils/src/testing/node/configuration/node_config_builder.rs#L43

https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-testing-utils/src/testing/node/configuration/node_config_builder.rs#L124

Legacy config tests:
https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-testing-utils/src/testing/node/configuration/legacy/configuration_builder.rs#L86

https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-testing-utils/src/testing/node/configuration/legacy/configuration_builder.rs#L145

https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-testing-utils/src/testing/node/configuration/legacy/config.rs


Hope this helps. But definitely basic configuration examples will be provided - thanks to mention this!

**Comment by eugene-babichenko** [comment]
@andrewarrow 

Thanks for reporting this. The documentation you provided needs to be updated.

Also this is documentation from the `master` branch which is not released yet. You might be interested in going back to the latest release tag.

**Comment by mzabaluev** [comment]
Fixed in #3237.
